src/github.com/matrix-org/dendrite/clientapi/routing/register.go

```diff
@@ -50,26 +50,34 @@ const (
 	sessionIDLength   = 24
 )
 
+// sessionsDict represents a session's completed flow stages
```

Comment:
does it? it looks like it tracks all sessions' data, not just one.

Reply:
Oh, my bad. Correcting.